.github/workflows/test_all_plugins.yml

```diff
@@ -42,29 +42,6 @@ jobs:
           miniforge-version: latest
           use-mamba: true
 
-      - id: check_conda
```

Comment:
Can you clarify why we're removing these checks?

Reply:
Cause I'm doing all of the env building now in the test itself (using `isolated_plugin_env`) ... so it's not needed in the test setup anymore